tests/integration-tests-topologies/src/main/java/org/apache/bookkeeper/tests/containers/ChaosContainer.java

```diff
@@ -85,7 +85,6 @@ public void tailContainerLog() {
     }
 
     public String getContainerLog() {
```

Comment:
This might hides a bug? After refactor, it seems meaningless.

Reply:
Indeed, the nethod was unused. I can remove it if you agree